packer/gpu/scripts/install-nvidia-docker.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -eu -o pipefail
+
+# Install Nvidia driver
+sudo yum install -y kernel-devel-$(uname -r) kernel-headers-$(uname -r) dkms # Running as ec2-user so need to sudo
```

Comment:
nit: move the `ec2-user` comment to script head? as it applies to the entire script (not just this line).

Reply:
done